### DISCUSSION TITLE
Support for AWS Dynamo DB

### DISCUSSION THREAD

**Discussion by evanlong-me** [prisma/prisma#25085] [Q&A]
### Question

docs:

https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Introduction.html

**Comment by ludralph** [comment]
Hi @evanlong0803 👋 

Thank you for raising this question.

We have an open [feature request](https://github.com/prisma/prisma/issues/1676) to support AWS DynamoDB. 

You can add your usecase and leave a 👍 on the issue to help our engineers prioritize it.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
